java/runtime/src/main/java/io/ray/runtime/metric/Count.java

```diff
@@ -22,6 +22,10 @@ public Count(String name, String description, String unit, Map<TagKey, String> t
     Preconditions.checkState(metricNativePointer != 0, "Count native pointer must not be 0.");
   }
 
+  public Count(String name, String description, Map<String, String> tags) {
```

Comment:
Should we remove  https://github.com/ray-project/ray/pull/17811/files#diff-2ffdfba93c661a2ab4ac6a1b23ef5b4a971acb8bf7ac020c6a88fcdc48098fd8R13 ?

Btw if we remove them, this PR will be an API change. Maybe we can mark the old API as deprecated?

Reply:
sounds great. I will add this annotation later.